third_party/pyth/evm-watcher/src/watchers/logger.ts

```diff
@@ -0,0 +1,11 @@
+import { Handler, PythUpdateEvent } from "../handler";
+
+export class Logger {
+  constructor(handler: Handler) {
+    handler.subscribe(this.processEvent.bind(this));
```

Comment:
imo this structure is confusing because logger references handler and handler references logger. Would suggest making logger an instance of the type that `subscribe` accepts and call handler.subscribe in the entrypoint script.

Reply:
Yeah it makes it more clear, will do.